radio/src/gui/128x64/model_setup.cpp

```diff
@@ -1139,7 +1139,7 @@ void menuModelSetup(event_t event)
               if (s_editMode > 0) {
                 if (l_posHorz == 1) {
                   if (IS_MODULE_R9M(moduleIdx) || (IS_MODULE_XJT(moduleIdx) && g_model.moduleData[moduleIdx].rfProtocol== RF_PROTO_X16)) {
-                    if (EVT_KEY_MASK(event) == KEY_ENTER) {
```

Comment:
@3djc we introduced this for the X-Lite, no?

Reply:
Yes we did, xlite isn't going into bind with ` if (event == EVT_KEY_BREAK(KEY_ENTER))`, we should at least #ifdef it

Reply:
Possibly some issue with the X-Lite navigation? Working differently from all others?

Reply:
Yes, very different from all Taranis/Horus, more like 9x (but we made it more ‘taranis’ like than plain 9x)